[Android] Permission Denial: startForeground requires android.permission.FOREGROUND_SERVICE

```
10-13 22:02:46.662 E/BOINC_GUI(28306):
10-13 22:02:46.662 E/BOINC_GUI(28306): at
android.app.IActivityManager$Stub.onTransact$setServiceForeground$(IActivityManager.java:11409)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
com.android.server.am.ActivityManagerService.setServiceForeground(ActivityManagerService.java:251
04)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
com.android.server.am.ActiveServices.setServiceForegroundLocked(ActiveServices.java:969)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
com.android.server.am.ActiveServices.setServiceForegroundInnerLocked(ActiveServices.java:1289)

10-13 22:02:46.662 E/BOINC_GUI(28306): at
com.android.server.am.ActivityManagerService.enforcePermission(ActivityManagerService.java:12335)
10-13 22:02:46.662 E/BOINC_GUI(28306): Caused by: android.os.RemoteException: Remote stack trace:
10-13 22:02:46.662 E/BOINC_GUI(28306): at java.util.TimerThread.run(Timer.java:512)
10-13 22:02:46.662 E/BOINC_GUI(28306): at java.util.TimerThread.mainLoop(Timer.java:562)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
edu.berkeley.boinc.client.Monitor$StatusUpdateTimerTask.run(Monitor.java:368)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
edu.berkeley.boinc.client.Monitor.access$200(Monitor.java:77)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
edu.berkeley.boinc.client.Monitor.updateStatus(Monitor.java:393)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
edu.berkeley.boinc.client.Monitor.readClientStatus(Monitor.java:464)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
edu.berkeley.boinc.client.ClientNotification.update(ClientNotification.java:146)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
edu.berkeley.boinc.client.ClientNotification.setForegroundState(ClientNotification.java:153)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
android.app.Service.startForeground(Service.java:695)
10-13 22:02:46.662 E/BOINC_GUI(28306): at
android.app.IActivityManager$Stub$Proxy.setServiceForeground(IActivityManager.java:5048)
10-13 22:02:46.662 E/BOINC_GUI(28306): at android.os.Parcel.readException(Parcel.java:1884)
10-13 22:02:46.662 E/BOINC_GUI(28306): at android.os.Parcel.readException(Parcel.java:1934)
10-13 22:02:46.662 E/BOINC_GUI(28306): at android.os.Parcel.createException(Parcel.java:1966)
10-13 22:02:46.662 E/BOINC_GUI(28306): java.lang.SecurityException: Permission Denial:
startForeground from pid=28306, uid=10226 requires android.permission.FOREGROUND_SERVICE
```

--- a/android/BOINC/app/src/main/AndroidManifest.xml
+++ b/android/BOINC/app/src/main/AndroidManifest.xml
@@ -34,6 +34,7 @@
     <uses-permission android:name="android.permission.WAKE_LOCK"/>
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
     <uses-permission android:name="android.permission.KILL_BACKGROUND_PROCESSES"/>
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
 
     <!-- Features required for Android TV, consoles, and set-top boxes like Nexus Player, OUYA,
          Razer Forge TV, Nvidia SHIELD, etc -->

